pkgs/by-name/bi/bisq2/package.nix

```
@@ -162,6 +167,21 @@ stdenvNoCC.mkDerivation rec {
     runHook postInstall
   '';
 
+  # The bisq2.webcam-app package is for maintainers to test scanning QR codes.
+  passthru.webcam-app = import ./webcam-app.nix {
```

this allows you to omit the inputs already in `pkgs`, you can then provide just the ones defined here
```suggestion
  passthru.webcam-app = callPackage ./webcam-app.nix {
```